### PR TITLE
Fixed screenshot fetchSize, removed defaults from XML

### PR DESCRIPTION
**Pull request by hannanelson**
Added missing source for other images, added missing .png images, added missing elements to Connector Reference

[24-after-Anypoint MQ Subscriber Source - Mule 4 _ MuleSoft Documentation.pdf](https://github.com/mulesoft/docs-connectors/files/5388536/24-after-Anypoint.MQ.Subscriber.Source.-.Mule.4._.MuleSoft.Documentation.pdf)
[25-after-Anypoint MQ Connector Reference - Mule 4 _ MuleSoft Documentation.pdf](https://github.com/mulesoft/docs-connectors/files/5388539/25-after-Anypoint.MQ.Connector.Reference.-.Mule.4._.MuleSoft.Documentation.pdf)

### CODE REVIEW

**Review by raju-sfdc — DISMISSED** [review]

File: anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-connector-reference.adoc [hunk 1]
```diff
@@ -509,20 +509,64 @@ Anypoint MQ Subscriber Message Source retrieves messages from the given destinat
 | Message Id a| String | ID of the message that was published |  | x
 |===
 
+[[prefetch]]
+=== Prefetch
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Max Local Messages a| Number | Maximum number of messages to be prefetched from the service and kept locally before being dispatched to the flow | 30 |
```
Comment (sarathecoubian) [inline anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-connector-reference.adoc]: "before messages are dispatched "

(removes passive voice)

File: anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-listener.adoc [hunk 2]
```diff
@@ -38,82 +32,87 @@ When using the `prefetch` mode, the `acknowledgementTimeout` timer begins from t
 
 == Predictable Message Consumption
 
-If your app requires predictable and controlled message consumption rather than maximum throughput, you can configure `Subscriber` as a polling source. The Subscriber source checks for new messages in the queue at a fixed scheduling rate. Every time the poll is triggered, the Subscriber source retrieves up to 10 messages to be dispatched to the flow in order, one by one. To do so, configure `polling` in `subscriber-type`.
+If your app requires predictable and controlled message consumption rather than maximum throughput, you can configure the Subscriber as a polling source. The Subscriber source checks for new messages in the queue at a fixed scheduling rate. Every time the poll is triggered, the Subscriber source retrieves between 1 and 10 messages to dispatch to the flow individually in order.
```
Comment (sarathecoubian) [inline anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-listener.adoc:35]: The term presentation guide usually takes issue with capitalizing non-product names. Is it possible in this context to keep Subscriber lower case?

Comment (sarathecoubian) [reply anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-listener.adoc:35]: I see later instances with Subscriber source, which would fit the standard, but reading this, I see why you've left it upper case.

File: anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-listener.adoc [hunk 3]
```diff
@@ -224,18 +223,20 @@ Use a Global Circuit Breaker when you want to share the circuit state across mul
...
 In the configuration wizard, populate the following fields as needed. Once the configuration is complete, you can reference this `Circuit_breaker` declaration from any Anypoint MQ Subscriber.
 
 To reference a circuit breaker:
 
 . Select the Subscriber source in the canvas.
 . Click the *Advanced* tab.
-. Select *Circuit Breaker > Global Reference*, and then choose one from the list.
-
-image::amq-3x-cb-global-ref.png[]
+. Select *Circuit Breaker > Global Reference*, and then choose a global circuit breaker configuration from the list.
```
Comment (sarathecoubian) [inline anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-listener.adoc]: and then select a global circuit breaker

**Review by sarathecoubian — DISMISSED** [review]
Looks good! Minor nitpicks to mull over.

File: anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-listener.adoc (hunk 2 above)
Comment (hannanelson) [reply anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-listener.adoc:35]: Subscriber is an operation - I just checked the SG and operations are init cap.

Comment (hannanelson) [reply anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-listener.adoc:35]: Subscriber is an operation - I just checked the SG and operations are init cap.

File: anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-connector-reference.adoc (hunk 1 above)
Comment (hannanelson) [reply anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-connector-reference.adoc]: Revised to remove all passive voice!

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**